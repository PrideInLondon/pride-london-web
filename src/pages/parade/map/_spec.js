import React from 'react'
import { shallow } from 'enzyme'
import ParadeMap from '.'

describe(ParadeMap.name, () => {
  it('render correctly', () => {
    const wrapper = shallow(<ParadeMap />)
    expect(wrapper).toMatchSnapshot()
  })
})
