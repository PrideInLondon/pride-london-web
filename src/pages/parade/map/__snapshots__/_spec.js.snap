// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ParadeMap render correctly 1`] = `
<Fragment>
  <map__ParadeMapContainer
    mx={0}
  >
    <Column
      pb={0}
      pl={
        Array [
          20,
          20,
          20,
          30,
          90,
        ]
      }
      pr={20}
      pt={
        Array [
          60,
          60,
          0,
        ]
      }
      px={null}
      py={null}
      width={
        Array [
          1,
          1,
          0.5,
          0.45,
          0.4,
        ]
      }
    >
      <map__ParadeMapContent>
        <div
          style={
            Object {
              "marginTop": "auto",
            }
          }
        >
          <map__Title>
            Parade map
          </map__Title>
          <map__Subtitle>
            Use it to find the best places to watch the parade and navigate to the different stages & areas.
          </map__Subtitle>
        </div>
        <map__BottomSection>
          <map__DownloadAppLabel>
            Download our iOS & Android app to carry the map around with you, internet connection is poor on the day.
          </map__DownloadAppLabel>
          <map__AppDownloadButtons>
            <map__AppDownloadButton
              aria-label="Download from App Store"
              href="https://itunes.apple.com/gb/app/pride-in-london/id1250496471"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                src=""
              />
            </map__AppDownloadButton>
            <map__AppDownloadButton
              aria-label="Download from Google Play"
              href="https://play.google.com/store/apps/details?id=org.prideinlondon.festival&hl=en"
              rel="noopener noreferrer"
              target="_blank"
            >
              <img
                src=""
              />
            </map__AppDownloadButton>
          </map__AppDownloadButtons>
        </map__BottomSection>
      </map__ParadeMapContent>
    </Column>
    <Column
      pb={0}
      pl={null}
      pr={null}
      pt={
        Array [
          0,
        ]
      }
      px={0}
      py={null}
      width={
        Array [
          1,
          1,
          0.5,
          0.55,
          0.6,
        ]
      }
    >
      <map__Map
        allowFullScreen={true}
        frameBorder="0"
        src="https://www.google.com/maps/d/embed?mid=1I5M9Oyc1damFcMhsi6yfODAMD6NXeIxi&z=16&ll=51.51004%2C%20-0.13501&hl=en&output=embed&t=m"
      />
    </Column>
  </map__ParadeMapContainer>
</Fragment>
`;
